mod/05_email/format/email_address_format.rb

```diff
@@ -0,0 +1,15 @@
+# -*- encoding : utf-8 -*-
+
+class Card::EmailAddressFormat < Card::TextFormat  
+  def link_to  text, href, opts={}
+    if text and href != text
+      "#{text} <#{href}>"
+    else
+      href
+    end
+  end
+
+  def chunk_list
+    :references
+  end
+end
```

Comment:
Can't we just make the external link recognizer for [[]] syntax also match without the mailto: ? It also begs the question of just what URL forms we recognize there, maybe that needs to be expanded to anything recognized as a free link.